src/components/checkboxFilter/index.tsx

```diff
@@ -38,6 +38,7 @@ type Props<ItemKey extends string> = {
    */
   onApply: (updatedItem: Item<ItemKey>, newState: State<ItemKey>) => void;
   numberOfColumnsOnDesktop?: number;
+  color?: ColorProps['textColor'];
```

Comment:
is there a requirement to make it configurable? I think it should be the same through all the application. In that case I'd just hardcode it

Reply:
You are right, no requirement to make it configurable. Removed it.